app/services/proofing/lexis_nexis/ddp/response_whitelister.rb

```diff
@@ -0,0 +1,31 @@
+module Proofing
+  module LexisNexis
+    module Ddp
+      class ResponseWhitelister
+        WHITELISTED_RESPONSE_FIELDS = %w[
```

Comment:
Let's call this an allowlister (or something) class and allowed fields
```suggestion
      class ResponseRedacter
        ALLOWED_RESPONSE_FIELDS = %w[
```